packages/@lwc/jest-preset-ssr/package.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "@lwc/jest-preset-ssr",
+    "version": "11.4.0",
+    "description": "Jest preset configuration and stubs to help test SSR rendering on LWC",
+    "homepage": "https://lwc.dev/",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/salesforce/lwc-test.git",
+        "directory": "packages/@lwc/jest-preset-ssr"
+    },
+    "bugs": {
+        "url": "https://github.com/salesforce/lwc-test/issues"
+    },
+    "license": "MIT",
+    "main": "jest-preset.js",
+    "keywords": [
+        "jest",
+        "lwc",
+        "ssr",
+        "server"
+    ],
+    "files": [
+        "/src/*.js"
+    ],
+    "peerDependencies": {
+        "@lwc/compiler": "^2",
+        "@lwc/engine-server": "^2",
+        "jest": "^26 || ^27 || ^28"
```

We [discussed](https://salesforce-internal.slack.com/archives/G0210EW0V2R/p1658425992729429) plans to drop support for jest26 for `@lwc/jest-preset`. Since this is a new package, jest26 should be skipped for it.
```suggestion
        "jest": ^27 || ^28"
```

---

Should we do that in a separate PR? It seems orthogonal to this PR, and would requite updating the Circle CI tests as well.

---

Sounds good.